pkg/test/framework/components/istio/kube.go

```diff
@@ -127,11 +127,19 @@ func (i *istioImpl) Ingresses() ingress.Instances {
 
 func (i *istioImpl) IngressFor(c cluster.Cluster) ingress.Instance {
 	ingressServiceName := defaultIngressServiceName
+	ingressServiceNamespace := i.cfg.SystemNamespace
+	ingressServiceLabel := defaultIngressIstioLabel
 	if serviceNameOverride := i.cfg.IngressGatewayServiceName; serviceNameOverride != "" {
```

Comment:
nit: should we default IngressGatewayServiceName in the cfg processing so we don't have to handle defaults everywhere?

Reply:
The rationale for the defaulting was in [this comment](https://github.com/istio/istio/pull/43809#discussion_r1130216781). Either way works, but I think I have a _slight_ preference for explicit defaults at this stage since not every test has an `istio.Instance` in its context (and therefore this default can't be relied on everywhere)